iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/udf/service/UDFExecutableManager.java

```diff
@@ -53,6 +58,47 @@ public static synchronized UDFExecutableManager setupAndGetInstance(
     return INSTANCE;
   }
 
+  /** check whether local jar is correct according to md5 */
+  public boolean isLocalJarConflicted(UDFInformation udfInformation) throws UDFManagementException {
+    String functionName = udfInformation.getFunctionName();
+    // A jar with the same name exists, we need to check md5
+    String existedMd5 = "";
+    String md5FilePath = functionName + ".txt";
```

Comment:
`.txt` already used previously? if not, I think `.md5` suffix may be better

Reply:
> `.txt` already used previously? if not, I think `.md5` suffix may be better

Yes. The code is copied from [UDFManagementService.java](https://github.com/apache/iotdb/pull/14135/files/656dc4c48bf6c3acd20f215729f7616aca636c0f#diff-f9da4fa77c651fa972a95aa9ae795f2dfb96a795d5cdd7e0fd5022be23537f0e)